OpenRA.Game/Traits/Player/FrozenActorLayer.cs

```diff
@@ -64,6 +64,18 @@ public FrozenActor(Actor self, PPos[] footprint, Shroud shroud, bool startsRevea
 				.Where(m => shroud.Contains(m))
 				.ToArray();
 
+			if (Footprint.Length == 0)
+				throw new ArgumentException(("This frozen actor has no footprint! Please report a bug!\n" +
+					"Map: {0}\nActor Location: {1}\n" +
+					"Input footprint: [{2}]\n" +
+					"Input footprint (shroud.Contains): [{3}]\n" +
+					"Input footprint (map.Contains): [{4}]")
+					.F(actor.World.Map.Title,
```

Comment:
This is probably redundant as we add it to every stacktrace, but the actor name would be interesting. I assume this could be bogus actor definition which we then could catch with lint checks.